internal/kafka/command_topic_update.go

```diff
@@ -114,11 +116,39 @@ func (c *command) update(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
-		updateResp, err := kafkaREST.CloudClient.UpdateKafkaTopicPartitionCount(topicName, kafkarestv3.UpdatePartitionCountRequestData{PartitionsCount: int32(updateNumPartitionsInt)})
+		_, err = kafkaREST.CloudClient.UpdateKafkaTopicPartitionCount(topicName, kafkarestv3.UpdatePartitionCountRequestData{PartitionsCount: int32(updateNumPartitionsInt)})
 		if err != nil {
 			return err
 		}
-		configsValues[numPartitionsKey] = fmt.Sprint(updateResp.PartitionsCount)
+
+		errPartition := retry.Retry(time.Second/10, time.Second, func() error {
```

Comment:
Is 1 second long enough for the partition count to finish updating?

Reply:
yes it should be good. I had tried and tested this in the past to reach this value